### DISCUSSION TITLE
the "init" fetch() parameter needs to be renamed

### DISCUSSION THREAD

**Discussion by AMDphreak** [mdn/mdn-community#556] [Content]
The "init" fetch() parameter is not descriptive of how the downstream programmer uses that field. It was named after what the upstream library/API developer was doing with the parameter in his/her code.

For clarity's sake, the "init" argument should be renamed to something more descriptive, like "settings". I mean, read the definition: it literally tells you what it should be named:
![image](https://user-images.githubusercontent.com/1058978/177086971-0d6e61c3-8b5b-4da9-84e8-4ebbba56bba6.png)

**Comment by MrBrain295** [comment]
This should be in mdn/community.

**Reply by ghost** [reply]
Could've also been an issue...

**Reply by teoli2003** [reply]
@AMDphreak Can you mark this discussion as resolved by selecting the answer below? This would have in showing that this discussion is closed.

Thank you!

**Comment by teoli2003** [comment]
We usually name these parameters `options`.

(Usually, prefer opening an issue to report a problem)

**Reply by teoli2003** [reply]
I opened #18330 to fix this.
